### PR TITLE
Gears Of War and 25 To Life ASLs.xml

### PR DESCRIPTION
**Pull request by LeonSReckon**
2 AutoSplitters at once I hope that works Okay!

If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [X] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [X] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [x] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

**Comment by just-ero** [comment]
Please fill out the full check list.

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -19505,4 +19505,32 @@
...
+            <Game>Gears of War for Windows</Game>
+            <Game>Gears of War GFWL</Game>
+            <Game>Gears of War Game For Windows Live</Game>
+            <Game>Gears of War 2006</Game>
+            <Game>Gears of War (2006)</Game>
+            <Game>Gears of War 1</Game>
+            <Game>Gears of War: 1</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/LeonSReckon/Gears-of-War-1-ASL/main/Gears%20Of%20War%201.asl?token=GHSAT0AAAAAACHPAJJXB65VUE2NKIWVKFIMZIACYFA</URL>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:19531]: This link leads to a 404 error.

Comment (LeonSReckon) [reply LiveSplit.AutoSplitters.xml]: I filled the check list
and I changed the link. For some reason this raw specifically creates a weird link

Comment (LeonSReckon) [reply LiveSplit.AutoSplitters.xml]: The link stopped working again I think the ASL/Repository file has a problem

Comment (just-ero) [reply LiveSplit.AutoSplitters.xml]: Is the repository not public?

Comment (LeonSReckon) [reply LiveSplit.AutoSplitters.xml]: I fixed the problem
I created a new repository

**Pull request merged**